plugins/com.google.cloud.tools.eclipse.appengine.deploy.ui/src/com/google/cloud/tools/eclipse/appengine/deploy/ui/DeployPropertyPage.java

```diff
@@ -202,7 +202,7 @@ private void evaluateFacetConfiguration() {
   private void updatePageBasedOnFacetConfig(boolean hasCorrectFacetConfiguration) {
     if (hasCorrectFacetConfiguration) {
       canSetMessage = true;
-      setErrorMessage(null);
+      content.getDataBindingContext().updateTargets();
```

Comment:
For the records: to reproduce https://github.com/GoogleCloudPlatform/google-cloud-eclipse/pull/880#issuecomment-256149028:

1. checkout `ca7bc9d90e0329ee204850e5e627e2ac698f8ed6`
1. launch the plugin
1. create a new GAE Standard project
1. Right click on the project, select Properties > Google Cloud Platform > App Engine Deployment
1. Click on the `Override generated version` checkbox (to mark it set)
1. Depending on the GTK version you use:
    1. On GTK2 the error message is visible instead of the title of the page, the page is invalid
    1. On GTK3 the title of the page remains visible, but the page is invalid because the Apply/OK buttons are disabled and switching to another preference page shows an error, that there are invalid values on the page